.golangci.yml

```diff
@@ -32,11 +37,14 @@ issues:
       linters:
         - goconst
         - unparam
-    # TODO: remove this when upgrading to golangci-lint >1.24.0.
-    # currently it checks the receiver names of even generated files, but is fixed in a later version
     - linters:
-        - stylecheck
-      text: 'ST1016: methods on the same type should have the same receiver name .*'
+        - gosec
+      # this was added in https://github.com/securego/gosec/issues/438 but for our use case is more or less entirely false positives
+      text: 'G601: Implicit memory aliasing in for loop.'
+    - linters:
+        - gosec
+      # TODO: we set some files to 0644, but I think we read them in the same process so 0600 might work?
```

Comment:
I could use some advice here, we use 644 in the deployer but I'm not sure if it's necessary

Reply:
`0600` should be fine as for CI everything is executed in the ci container as root.